pkg/get.go

```diff
@@ -120,3 +120,20 @@ func GetAppNameAndIP(appLabel, appNS string, clients environment.ClientSets) (st
 	return PodList.Items[0].Name, PodList.Items[0].Status.PodIP, PodList.Items[1].Name, nil
 
 }
+
+// GetUID will return the uid from experiment label
+func GetUID(engineName, namespace string, clients environment.ClientSets) (error, string) {
+
+	chaosEngine, err := clients.LitmusClient.ChaosEngines(namespace).Get(engineName, metav1.GetOptions{})
+	if err != nil {
+		return errors.Errorf("fail to get the chaosengine %v err: %v", engineName, err), ""
+	}
+	if len(chaosEngine.Status.Experiments) == 0 {
+		return errors.Errorf("fail to get the chaos pod for the test"), ""
+	}
+	chaosPod, err := clients.KubeClient.CoreV1().Pods(namespace).Get(chaosEngine.Status.Experiments[0].ExpPod, metav1.GetOptions{})
+	if err != nil {
+		return errors.Errorf("fail to get the chaos pod. err: %v", err), ""
+	}
+	return nil, chaosPod.Labels["chaosUID"]
```

Comment:
We can directly extract the UID from the engine right?

Reply:
Yes right, Have updated!